src/utils/globalJWInterface.ts

```diff
@@ -56,7 +56,7 @@ export const getCurrentQuality = withTryCatchDefault(() => {
 });
 
 export const getIsCurrentlyViewable = withTryCatchDefault(() => {
-	return !!window.jwplayer().getViewable();
+	return !!window.jwplayer().getViewable() ?? false;
```

Comment:
Minor nitpick, but wouldn't `??` be redundant here ?

Since the API returns `1` or `0`, `!!` will convert the appropriate value to a boolean and handle `null` and `undefined` accordingly

Reply:
Discussed offline and it's some odd issue and this seemed to fix it 🤷